include/ChocAn/app/state_viewer.hpp

```diff
@@ -22,6 +22,7 @@ Authors: Daniel Mendez
 #include <memory>
 #include <functional>
 #include <ChocAn/app/application_state.hpp>
+#include <ChocAn/core/utils/account_builder.hpp>
```

Comment:
The interface shouldn't need this include.

Reply:
its gone now